### PR TITLE
feat: main page

### PR DESCRIPTION
**Pull request by heedong12**
## Docs

- [figma design template](https://www.figma.com/design/TJmCclkBRGK4iZ9pLPbvgw/Maeum?node-id=0-1&t=4FJySNo4Vttv3vGb-0)

## Changes

- before :
empty page
- after :
publishing main page
move to next page

- images
image

## Review Points
- initial settings
- url 

#### Problem
how to define url

#### Solution
use detail page urls to '/script/~'

## Test Checklist

- [ ] check 1 : main page prototype

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <link rel="icon" href="/favicon.ico">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Vite App</title>
+    <meta charset="UTF-8" />
+    <link rel="icon" href="/favicon.ico" />
```
Comment (kim-song-jun) [inline index.html]: No icon?

File: index.html [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>MAEUM</title>
```
Comment (kim-song-jun) [inline index.html]: How about change to `Maeum`?

File: src/components/MainPage.vue [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  <div
+    class="h-full flex flex-col justify-center items-center bg-main_background bg-no-repeat bg-bottom"
+  >
+    <div class="flex flex-col justify-center items-center flex-1">
+      <p class="font-semibold text-primary">스크립트를 마음에 담다</p>
+      <p class="text-4xl text-primary font-extrabold tracking-wide">MAEUM</p>
+      <img src="../../public/image/main_logo.png" class="w-5/6" />
+    </div>
+    <button
+      class="my-11 h-14 w-11/12 bg-white rounded-2xl text-button_color font-bold"
```
Comment (kim-song-jun) [inline src/components/MainPage.vue]: Do not use `snake_case` in class name. use `-` instead of `_`

File: src/components/MainPage.vue [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+      @click="goNextPage"
```
Comment (kim-song-jun) [inline src/components/MainPage.vue]: Change function name
```js
handleClickStartButton() {...}
```

File: src/components/MainPage.vue [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    >
+      시작하기
+    </button>
+  </div>
+</template>
+
+<script>
+export default {
+  methods: {
+    goNextPage() {
```
Comment (kim-song-jun) [inline src/components/MainPage.vue]: See above

File: src/components/MainPage.vue [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+      this.$router.push("/script/name");
```
Comment (kim-song-jun) [inline src/components/MainPage.vue]: What is `/script/name` mean?

Comment (kim-song-jun) [reply src/components/MainPage.vue]: Use meaningful url name

File: src/main.js [hunk 7]
```diff
@@ -1,11 +1,12 @@
...
+import { createApp } from "vue";
+import { createPinia } from "pinia";
+import App from "./App.vue";
+import router from "./router";
 
-const app = createApp(App)
+const app = createApp(App);
 
-app.use(createPinia())
+app.use(createPinia()).use(router);
```
Comment (kim-song-jun) [inline src/main.js]: Use const for pinia.

```js
const store = createPinia()
app.use(store).use(router).mout("#app");
```

File: tailwind.config.js [hunk 8]
```diff
@@ -0,0 +1,18 @@
+/** @type {import('tailwindcss').Config} */
+export default {
+  content: ["./index.html", "./src/**/*.{html,js,vue}"],
+  theme: {
+    extend: {
+      colors: {
+        primary: "#FF6900",
+        button_color: "#585858",
+        box_color: "#F4F4F4",
```
Comment (kim-song-jun) [inline tailwind.config.js]: See above. check class name.

File: tailwind.config.js [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+        select_box_color: "#FFE1E1",
```
Comment (kim-song-jun) [inline tailwind.config.js]: If you want to use color keyword in tailwind, must add status of color.

```js
colors:{
   primary-box: '#F4F4F4'
}
```

File: tailwind.config.js [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+      },
+      backgroundImage: {
+        main_background: "url(/image/main_background.png)",
```
Comment (kim-song-jun) [inline tailwind.config.js]: See above

**Review by kim-song-jun — COMMENTED** [review]
👍  Plz check review.

File: index.html (hunk 1 above)
Comment (heedong12) [reply index.html]: Is it means maeum icon?

Comment (kim-song-jun) [reply index.html]: Yes. Use main icon

File: index.html [hunk 11]
```diff
@@ -1,10 +1,10 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <link rel="icon" href="/favicon.ico">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Vite App</title>
+    <meta charset="UTF-8" />
+    <link rel="icon" href="/public//image/main-logo.png" />
```
Comment (kim-song-jun) [inline index.html]: typo? and change to images

**Review by kim-song-jun — COMMENTED** [review]
LGTM

**Review by kim-song-jun — COMMENTED** [review]
LGTM

**Pull request merged**